src/style.css

```diff
@@ -25,15 +25,11 @@ header {
   margin: auto;
   margin-top: 5rem;
   padding: 2.5rem;
-  padding-bottom: 0;
-  padding-left: 0;
-  padding-right: 0;
   border-radius: 0.5rem;
   width: 80%;
   display: flex;
   flex-direction: column;
-  border: 1px solid gold;
-  box-shadow: 0 20px 20px 4px;
+  border: 1px solid #494949;
```

Comment:
- [ ] Great job so far your remove and clear all completed functions are working fine. However, when a new task is added I can't see what is added it's not displayed on the list and just appear as blank also in the local storage the tasks are not visible.

https://user-images.githubusercontent.com/70269507/194902892-822a09de-7c99-4708-a5d3-56b7278dad49.mov

<img width="1193" alt="Screenshot 2022-10-10 at 6 28 18 PM" src="https://user-images.githubusercontent.com/70269507/194902906-252bbda9-b101-4b91-b82f-47d1bba09adc.png">